Fundamentals/Setup/Upgrading/version-specific.md

```diff
@@ -68,21 +66,20 @@ public class Program
                     webBuilder.UseStartup<Startup>();
                 });
     }
-```
-
-The calls to `ConfigureUmbracoDefaults` and `webBuilder.UseStaticWebAssets()` are new.
-
-Finally, remove the following files and folders:
-
-- `/wwwroot/umbraco`
-- `/umbraco/PartialViewMacros`
-- `/umbraco/UmbracoBackOffice`
-- `/umbraco/UmbracoInstall`
-- `/umbraco/UmbracoWebsite`
-- `/umbraco/config/lang`
-- `/App_Plugins/UmbracoForms` (if you are using Umbraco Forms on your project)
-
-To re-enable the appsettings IntelliSense, you must update your schema reference in the **appsettings.json** file from:
+    ```
+    The calls to `ConfigureUmbracoDefaults` and `webBuilder.UseStaticWebAssets()` are new.
+
+9. Remove the following files and folders:
+    - `/wwwroot/umbraco`
+    - `/umbraco/PartialViewMacros`
+    - `/umbraco/UmbracoBackOffice`
+    - `/umbraco/UmbracoInstall`
+    - `/umbraco/UmbracoWebsite`
+    - `/umbraco/config`
```

Comment:
```suggestion
    - `/umbraco/config/lang`
```

Reply:
OK, in that case I think we should also remove the old appsettings schema specifically.